<alloca.h> is implicitly included by <stdlib.h> on Linux

In file included from visa/iga/IGAExe/assemble.cpp:27:
In file included from visa/iga/IGAExe/iga_main.hpp:29:
visa/iga/IGAExe/fatal.hpp:33:10: fatal error: 'alloca.h' file not found
#include <alloca.h>
         ^~~~~~~~~~

visa/iga/IGALibrary/strings.cpp:41:10: fatal error: 'alloca.h' file not found
#include <alloca.h> /* for alloca */
         ^~~~~~~~~~

--- visa/iga/IGAExe/fatal.hpp.orig	2019-11-27 01:05:20 UTC
+++ visa/iga/IGAExe/fatal.hpp
@@ -30,7 +30,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include <Windows.h>
 #include <malloc.h>
 #else
-#include <alloca.h>
+#include <stdlib.h>
 #endif
 #ifdef _MSC_VER
 #define VSCPRINTF(PAT,VA) \
--- visa/iga/IGALibrary/strings.cpp.orig	2019-11-27 01:05:20 UTC
+++ visa/iga/IGALibrary/strings.cpp
@@ -38,7 +38,7 @@ using namespace iga;
 #include <Windows.h>
 #include <malloc.h> /* for alloca */
 #else
-#include <alloca.h> /* for alloca */
+#include <stdlib.h> /* for alloca */
 #endif
 
 std::string iga::format(const char *pat, ...)
